src/components/Pagination.js

```diff
@@ -0,0 +1,87 @@
+import styles from "./pagination.module.scss";
+import PaginationSpinner from "./PaginationSpinner";
+
+export default function Pagination({
+  numberOfPages,
+  currentPage,
+  setCurrentPage,
+  isLoading,
+}) {
+  const pageNumbers = [];
+  const PreviousPageNumber = currentPage - 1;
+  const NextPageNumber = currentPage + 1;
+
+  for (let i = 1; i <= numberOfPages; i++) {
+    if (i === numberOfPages || i === 1) {
+      pageNumbers.push(i);
+      continue;
+    }
+
+    if (i === currentPage || i === PreviousPageNumber || i === NextPageNumber) {
```

Comment:
This condition do the same thing as the previous one, so unite them please

Reply:
And try to do something like this `[numberOfPages, 1, currentPage ...].inlucde(i)` in the condition